packages/amplify-graphql-name-mapping-transformer/src/__tests__/__integ__/__snapshots__/with-has-many.test.ts.snap

```diff
@@ -247,9 +247,7 @@ exports[`@refersTo with @hasMany for RDS Models model table names are mapped 4`]
 #set( $lambdaInput.operationName = \\"getEmployee\\" )
 #set( $lambdaInput.args.metadata = {} )
 #set( $lambdaInput.args.metadata.keys = [] )
-#if( !$util.isNullOrEmpty($ctx.stash.authFilter) )
-  #set( $lambdaInput.args.metadata.authFilter = $ctx.stash.authFilter )
-#end
+#set( $lambdaInput.args.metadata.authFilter = {} )
```

Comment:
Do we have confidence that this change is safe? Are we sure that there will never be data in `$lambdaInput.args.metadata.authFilter`?

Reply:
In this case we want the authFilter to be cleared because we want the lambda to return the record to the resolver even it is not authorized. We are now using the `validateUsingSource` util function against the result to verify the authorization.

In this case there is information in the authFilter but we are shifting that information to `authRules` and `validateUsingSource`.